Source/WebCore/loader/cache/CachedFont.cpp

```diff
@@ -134,8 +129,16 @@ String CachedFont::calculateItemInCollection() const
 bool CachedFont::ensureCustomFontData(SharedBuffer* data)
 {
     if (!m_fontCustomPlatformData && !errorOccurred() && !isLoading() && data) {
-        bool wrapping;
-        m_fontCustomPlatformData = createCustomFontData(*data, calculateItemInCollection(), wrapping);
+        bool wrapping = false;
+        if (m_mustLoadWithMemorySafeParser) {
+            if (m_loader->frame()->settings().experimentalFontParserInLockdownMode())
```

Comment:
I think we should avoid the 'Experimental' name here as I mentioned in `UnifiedWebPreferences`